README.md

```diff
@@ -60,22 +60,39 @@ A third-party plugin is required to enable syntax coloring. The Che4z basic stac
 	
 ## Copybook Support
 
-The LSP for COBOL extension can retrieve copybooks used in your projects from the mainframe and download them locally. You can open copybooks in your IDE and make use of the copybook support features of the extension.
+The COBOL Language Support extension can retrieve copybooks used in your projects from the mainframe and download them locally. You can also store copybooks locally in folders in your workspace. Copybook support features of the extension are enabled whenever a copybook called in source code is successfully located in a specified mainframe data set or local folder.
```

Comment:
Add something like: "The COBOL Language Support extension requires all copybooks referred in the code after the 'COPY' statements to be present locally in order to avoid false positives in error check, goto definition etc. They could be either already part of the workspace (e.g. cloned from the GIT repository along with the whole project), added manually into some local folder, or downloaded from mainframe." Check with the team if the local copybooks can be in a diff folder than the current workspace? I don't think so as we support only relative paths now. Also mention that we cover only relative paths.

Reply:
I already clarified that local copybooks need to be in a workspace folder, and that paths need to be relative. Is this not clear enough from the current procedure under "Storing Copybooks Locally"?

Reply:
I've reworded the section opening, hope it's clearer now.